src/customJs/socialShareTool/SocialShareTool.js

```diff
@@ -3,6 +3,10 @@ import PropTypes from 'prop-types';
 
 export const SocialShareTool = (props) => {
   const values = props.values;
+  const { social_share_size: size } = props.values;
```

Comment:
Since we are destructuring in the second line, I would remove the first one to avoid having the same information with different names.

```suggestion
  const { social_share_size: size } = props.values;
```